x-pack/plugins/security/server/lib/authorization/service.js

```diff
@@ -20,6 +20,7 @@ export function createAuthorizationService(server, xpackInfoFeature) {
     actions,
     checkPrivilegesWithRequest,
     config,
+    (...args) => server.log(...args),
```

Comment:
we can't just pass a reference to the log function, it loses its `this` reference. We could also rebind it via `server.log.bind(server)`, but I thought this was cleaner

Reply:
I like this approach better.